config/initializers/mime_types.rb

```diff
@@ -0,0 +1 @@
+Mime::Type.register "application/vnd.ms-excel", :xls
```

Comment:
do not call this register if already registered in system
because main app already can have this